tests/data/test1130

```diff
@@ -52,7 +52,7 @@ http
 HTTP POST forced expect 100-continue with a 404
  </name>
  <command option="no-output">
--d @%LOGDIR/file%TESTNUMBER http://%HOSTIP:%HTTPPORT/%TESTNUMBER http://%HOSTIP:%HTTPPORT/%TESTNUMBER0001 -H "Expect: 100-continue" --expect100-timeout 999
+-d @%LOGDIR/file%TESTNUMBER http://%HOSTIP:%HTTPPORT/%TESTNUMBER http://%HOSTIP:%HTTPPORT/%TESTNUMBER0001 -H "Expect: 100-continue" --expect100-timeout 99
```

Comment:
why change these tests to have a shorter expect timeout? isn't the longer timeout more reliable?

Reply:
This is a) for consistency, and b) for expediency. If something goes horribly wrong and the timeout needs to be used to end the test, this will cause a 16 minute delay before the test exits, while 99 seconds is way more than enough already.